### DISCUSSION TITLE
v0.12.0-BETA.1

### DISCUSSION THREAD

**Discussion by adultmm** [adultmm/AdultMediaManager#74] [General]
Perhaps the most frequently requested feature is to implement downloading of actors' images from the sites. In this version, I developed it for most of the scrapers. This can slow down the scraping, or can download really tiny images, so I also implemented a new settings tab, where you can switch it off per Scraper: Edit > Settings... > Movies > Scrapers - Data > AdultScraper > Video scrapers

These changes have required many refactorings, so I release it as a beta version. 

- **ADD: video scrapers now tries to download actors' images from separated actor's pages**
- **ADD: possibility to edit some settings of VideoScrapers**
- ADD: BrokenLatinaWhores.com scraper

- ADD: AnalizedNetwork scraper now supports PornForce.com site

- ADD: you can disable any VideoScraper (Edit > Settings... > Movies > Scrapers - Data > AdultScraper > Video scrapers > Select the scraper to disable > Enabled: False)
- ADD: you can enable or disable any VideoScraper to download actors' images or not (Edit > Settings... > Movies > Scrapers - Data > AdultScraper > Video scrapers > Select the scraper to disable > Scrape Actors' Images : True/False)

- ADD: AmateurAllure now supports actors' images
- ADD: AnalizedNetwork scraper now support actors' images
- ADD: AnalVids scraper now support actors' images
- ADD: BangBros scraper now support actors' images
- ADD: BellaPass scraper now support actors' images
- ADD: CherryPimps scraper now support actors' images
- ADD: FrolicMe scraper now support actors' images
- ADD: GammaEntertainment, 21Sextury and ZeroTolerance sites now supports actors' images
- ADD: HookupHotspot scraper now supports actors' images
- ADD: HussiePass scraper now supports actors' images
- ADD: JulesJordanNetwork scraper now support actors' images
- ADD: Kink scraper now support actors' images
- ADD: LetsDoeIt scraper now support actors' images
- ADD: MissaX scraper now support actors' images
- ADD: NaughtyAmerica scraper now support actors' images
- ADD: PinkVisualPass scraper now support actors' images
- ADD: PlumperPass scraper now support actors' images
- ADD: PornWorld scraper now support actors' images
- ADD: PrivateNetwork scraper now supports actors' images
- ADD: ScoreUniverse scraper (Scoreland, BigBoobsPOV, etc) now support actors' images
- ADD: TukTukPatrol scraper now tries to download actors' images from the site if the actor's link exists
- ADD: X-Art scraper now support actors' images

- ADD: 1Pondo and 10Musume scrapers - AddSeriesAsTags and AddSeriesAsTagsPattern config
- ADD: Bang scraper - AddMovieInfoToPlot config

<br/>

- MOD: NewSensation scraper now uses ImageProxy
- MOD: remowed www. prefix from the source text of TrueAnal network sites

<br/>

- FIX: Data18 title has changed
- FIX: some site structure has changed in TrueAnalNetwork (AllAnal, AnalOnly, etc)

<br/>
<b> You can help the development with any small <a href="https://www.paypal.com/donate/?hosted_button_id=8AN7JHZZWTFQA" target="_blank">donation</a>. Thanks</b>
<br/><br/>
<a href="https://www.paypal.com/donate/?hosted_button_id=8AN7JHZZWTFQA" target="_blank"><img src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_SM.gif" alt="Support development of AdultMediaManager" style="height: 30px !important;width: 108px !important;"></a>&nbsp;&nbsp;&nbsp;&nbsp;

<br/><br/>

<hr /><em>This discussion was created from the release <a href='https://github.com/adultmm/AdultMediaManager/releases/tag/v0.12.0-BETA.1'>v0.12.0-BETA.1</a>.</em>

**Comment by TwisterSpa** [comment]
Hy , the source files seem to be empty files.

**Comment by wkearney99** [comment]
Any updates coming?  If not, any source?

**Reply by adultmm** [reply]
Hi, I started a huge refactor to make it possible to change the scraper settings for data and images separately. So I need some time to finish it, but I have a paying development job, so my free time is very limited. Sorry, it'll take some weeks for the next beta.

**Comment by adultmm** [comment]
Hi @wkearney99,

New version is available: [v0.12.0-BETA.2](https://github.com/adultmm/AdultMediaManager/releases/tag/v0.12.0-BETA.2)

**Comment by hotshot08** [comment]
Bundled version: Just download, extract and use.
EMM Plugin version: Extract it under your EmberMediaManager/Module folder, then enable AdultScraper under Edit/Settings/Movies.

I have follow the above the instructions but when I try to search for a title it gives me nothing, please help.
